src/main/java/com/cdancy/bitbucket/rest/features/RepositoryApi.java

```diff
@@ -82,4 +83,15 @@ boolean delete(@PathParam("project") String project,
     RepositoryPage list(@PathParam("project") String project,
                         @Nullable @QueryParam("start") Integer start,
                         @Nullable @QueryParam("limit") Integer limit);
+
+    @Named("repository:list-Permissions-Group")
+    @Documentation({"https://developer.atlassian.com/static/rest/bitbucket-server/5.0.0/bitbucket-rest.html#idm45659054969200"})
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Path("/{project}/repos/{repo}/permissions/groups")
+    @Fallback(BitbucketFallbacks.PermissionsPageOnError.class)
+    @GET
+    PermissionsPage listPermissionsGroup(@PathParam("project") String project,
```

Comment:
@j0nathan33 FYI ... I'm going to rename this on master to `listGroupPermissions`.